Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Apsitvarkom.Models;
+using Apsitvarkom.Models.DTO;
+using Microsoft.AspNetCore.Mvc;
+namespace Apsitvarkom.Api.Controllers;
```

Comment:
According to what we have been doing in other files, a new line would be preferred here before the namespace. That's up for a discussion if You think this way is better though, but we would ideally have it consistently.